UniAtHome/UniAtHome.BLL/Util/MappingProfile.cs

```diff
@@ -0,0 +1,15 @@
+﻿using AutoMapper;
+using UniAtHome.BLL.DTOs;
+using UniAtHome.DAL.Entities;
+
+namespace UniAtHome.BLL.Util
```

Comment:
Please, put automapper configuration in ./Configuration folder in the WebAPI project